### DISCUSSION TITLE
How can I create my own widget on the accounts -> summary page?

### DISCUSSION THREAD

**Discussion by blu-IT** [YetiForceCompany/YetiForceCRM#17084] [How to]
Hello,

I would like to customize the _summary widge_t and/or _create a new widget_ on the summary page of the accounts view.

I know that in _Software configuration -> Standard modules -> Modules - Widget_ the widgets for the summary page can be selected, but I want to create my own one or even have the possibility to edit existing the summary widget.

Is there a way?

Thanks!

**Comment by apcloic** [comment]
@blu-IT 
Maybe check this one :
#16643
